content/hi/docs/reference/glossary/cloud-provider.md

```diff
@@ -0,0 +1,22 @@
+---
+title: क्लाउड प्रदाता (Cloud Provider)
+id: cloud-provider
+date: 2018-04-12
+short_description: >
+  एक संगठन जो क्लाउड कंप्यूटिंग प्लेटफॉर्म प्रदान करता है।
+
+aka:
+  - क्लाउड सेवा प्रदाता (Cloud Service Provider)
+tags:
+  - community
+---
+
+एक व्यवसाय या अन्य संगठन जो क्लाउड कंप्यूटिंग प्लेटफॉर्म प्रदान करता हैं।
+
+<!--more-->
+
+क्लाउड प्रदाता, जिन्हें कभी-कभी क्लाउड सेवा प्रदाता (CSP) भी कहा जाता है, क्लाउड कंप्यूटिंग प्लेटफॉर्म या सेवाएं प्रदान करते हैं।
+
+कई क्लाउड प्रदाता प्रबंधित अवसंरचना प्रदान करते हैं (जिन्हें Infrastructure as a Service या IaaS भी कहा जाता है)। प्रबंधित अवसंरचना के साथ क्लाउड प्रदाता सर्वर, स्टोरेज और नेटवर्किंग प्रदान करने के लिए जिम्मेदार है जबकि आप उसके ऊपरी लेयर्स का प्रबंधन करते हैं जैसे कि  कुबेरनेट्स क्लस्टर चलाना।
+
+आप कुबेरनेट्स को एक प्रबंधित सेवा के रूप में भी पा सकते हैं; कई बार इसे Platform as a Service, या PaaS भी कहा जाता है। प्रबंधित कुबेरनेट्स के साथ, आपका क्लाउड प्रदाता कुबेरनेट्स कंट्रोल प्लेन के साथ-साथ {{< glossary_tooltip text="नोड" term_id="node" >}} और जिस अवसंरचना पर वे भरोसा करते हैं: नेटवर्किंग, स्टोरेज, और संभवतः अन्य तत्व जैसे लोड बैलेंसर्स के लिए जिम्मेदार है।
```

Comment:
Too many "साथ" in one sentence. Eliminated this to retain conciseness without taking away from the meaning.
```suggestion
आप कुबेरनेट्स को एक प्रबंधित सेवा के रूप में भी पा सकते हैं; कई बार इसे Platform as a Service, या PaaS भी कहा जाता है। प्रबंधित कुबेरनेट्स के साथ, आपका क्लाउड प्रदाता कुबेरनेट्स कंट्रोल प्लेन{{< glossary_tooltip text="नोड" term_id="node" >}} और जिस अवसंरचना पर वे भरोसा करते हैं: नेटवर्किंग, स्टोरेज, और संभवतः अन्य तत्व जैसे लोड बैलेंसर्स के लिए जिम्मेदार है।
```